src/test/scala/com/microsoft/hyperspace/index/rules/RuleUtilsTest.scala

```diff
@@ -110,6 +111,71 @@ class RuleUtilsTest extends HyperspaceRuleTestSuite {
     assert(r.isEmpty)
   }
 
+  test("Verify getCandidateIndex for hybrid scan") {
+    val indexManager = IndexCollectionManager(spark)
+    val df = spark.range(1, 5).toDF("id")
+    val dataPath = systemPath.toString + "/hbtable"
+    df.write.parquet(dataPath)
+
+    val readDf = spark.read.parquet(dataPath)
+    indexManager.create(readDf, IndexConfig("index1", Seq("id")))
```

Comment:
Can we use `withIndex` to clean up indexes created? (btw, I am thinking of a better way to to handle data path, so let's not worry about it for now).